package.json

```diff
@@ -102,108 +102,117 @@
+    "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sass": "^11.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-element-helper": "^0.8.6",
-    "ember-eslint-parser": "^0.3.8",
-    "ember-intl": "5.7.2",
+    "ember-eslint-parser": "^0.4.3",
+    "ember-intl": "~7.0.3",
     "ember-leaflet": "^5.1.3",
     "ember-load-initializers": "^2.1.2",
-    "ember-modifier": "3.2.7",
-    "ember-page-title": "^8.0.0",
-    "ember-power-select": "6.0.0",
-    "ember-qunit": "^6.0.0",
+    "ember-modifier": "~4.1.0",
+    "ember-page-title": "^8.2.3",
+    "ember-power-select": "~7.1.0",
+    "ember-qunit": "^6.2.0",
     "ember-resolver": "^11.0.1",
     "ember-source": "4.12.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.11.2",
+    "ember-template-lint": "^5.13.0",
     "ember-truth-helpers": "^4.0.3",
-    "ember-try": "^2.0.0",
-    "eslint": "^8.44.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-ember": "^12.0.2",
+    "ember-try": "^3.0.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^12.1.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^5.0.0",
-    "eslint-plugin-qunit": "^8.0.1",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-qunit": "^8.1.1",
     "leaflet": "^1.9.4",
     "loader.js": "^4.7.0",
-    "prettier": "^3.2.0",
-    "prettier-plugin-ember-template-tag": "^2.0.1",
+    "prettier": "^3.3.2",
+    "prettier-plugin-ember-template-tag": "^2.0.2",
     "prosemirror-dev-tools": "^4.1.0",
-    "qunit": "^2.19.2",
+    "qunit": "^2.21.0",
     "qunit-dom": "^2.0.0",
-    "release-it": "^15.5.0",
-    "rimraf": "^5.0.5",
-    "sass": "^1.49.7",
+    "release-it": "^16.3.0",
+    "rimraf": "^5.0.7",
+    "sass": "^1.77.6",
     "sinon": "^17.0.1",
     "typescript": "~5.3.3",
-    "webpack": "^5.74.0"
+    "webpack": "^5.92.1"
   },
   "peerDependencies": {
     "@appuniversum/ember-appuniversum": "^3.4.1",
     "@ember/string": "3.x",
-    "@glint/template": "^1.1.0",
+    "@glint/template": "^1.4.0",
     "@lblod/ember-rdfa-editor": "^9.13.0",
-    "ember-concurrency": "^2.3.7 || ^3.1.0",
+    "ember-concurrency": "^3.1.0",
     "ember-element-helper": "^0.8.6",
-    "ember-intl": "^5.7.2 || ^6.1.0",
+    "ember-intl": "^7.0.0",
     "ember-leaflet": "^5.1.3",
-    "ember-modifier": "^3.2.7 || ^4.0.0",
-    "ember-power-select": "6.x || 7.x",
+    "ember-modifier": "^4.0.0",
+    "ember-power-select": "^7.1.0",
     "ember-source": "^4.12.0",
+    "ember-template-imports": "^4.1.1",
     "ember-truth-helpers": "^4.0.3",
     "leaflet": "^1.9.4"
   },
-  "overrides": {
-    "ember-intl": {
-      "typescript": "^5.0.4"
+  "peerDependenciesMeta": {
+    "@glint/template": {
+      "optional": true
+    },
+    "ember-template-imports": {
```

Comment:
glint I get, but is e-t-i really optional? Doesn't it provide the build hooks to compile the gts files? remember this is still a v1 addon and as such is "compiled" by the app at app build time

Reply:
It's not really optional. An optional peer dep seems to be the only way to say "if you install this, it needs to satisfy these version constraints". Since it's still in the dependencies rather than being moved to dev dependencies, it'll always be installed, in npm/yarn *somewhere* in node_modules and in pnpm nested under lblod/plugins. It does feel weird to put it here, but since there's no way to write comments in the package.json, I can't explain it in the file...

Reply:
Yeah it's weird indeed, I guess this is why https://github.com/salsify/ember-cli-dependency-lint was created.

Reply:
"if you install this, it needs to satisfy these version constraints" -> I was under the impression that this is what peerdeps did already? 

you'd remove it from the deps and only add it peerdeps in this case.

Reply:
yeah but I thought that's what peerdeps are for

you add it to devdeps and peerdeps so:
- client does not depend: its automatically installed by any version of npm we still support
- client does depend: version clashes are errors, and if no clash, the client's version is used
- building the test app: the devdep is used

Reply:
note that's my understanding of how peerdeps work, I'm fully open to the possibility I'm wrong

Reply:
The choice of deps rather than peerdeps is down to Ember (I assume just for v1 addons). It treats production dependencies differently to dev dependencies. Simply moving e-t-i to devdeps and leaving nothing else unchanged leads to a runtime error when trying to use a component defined in a gts file. Otherwise, I agree that it should be a devdep as it should only be needed at compile time.

Reply:
oh wow yeah hadn't considered ember throwing a wrench in the works